src/Raven.Server/Documents/Replication/ResolveConflictOnReplicationConfigurationChange.cs

```diff
@@ -161,6 +163,21 @@ public override int Execute(DocumentsOperationContext context)
 
                 return count;
             }
+
+            public override TransactionOperationsMerger.IReplayableCommandDto<TransactionOperationsMerger.MergedTransactionCommand> ToDto(JsonOperationContext context)
+            {
+                // The LowerId created as in memory LazyStringValue
+                // so EscapePositions set to empty to avoid reference to escape bytes (after string bytes) while serializing
+                foreach (var conflict in _resolvedConflicts)
+                {
+                    conflict.ResolvedConflict.LowerId.EscapePositions = Array.Empty<int>();
```

Comment:
Don't we need to make a copy of them then?

Reply:
change the comment, yes